dev/benchmark/sysbench-in-k8s.md

```diff
@@ -0,0 +1,446 @@
+---
+title: TiDB in Kubernetes Sysbench 性能测试
+category: benchmark
+---
+
+# TiDB in Kubernetes Sysbench 性能测试
+
+随着 [TiDB Operator GA 发布](https://pingcap.com/blog-cn/tidb-operator-1.0-ga/)，越来越多用户开始使用 TiDB Operator 在 Kubernetes 中部署管理 TiDB 集群。在本次测试中，我们选择 GKE 平台做了一次深入、全方位的测试，方便大家了解 TiDB 在 Kubernetes 中性能影响因素。
+
+## 目的
+
+- 测试典型公有云平台上 TiDB 性能数据
+- 测试公有云平台磁盘、网络、CPU 以及不同 Pod 网络下对 TiDB 性能的影响
+
+## 环境
+
+### 版本与配置
+
+本次测试统一使用 TiDB v3.0.1 版本进行测试。
+
+TiDB Operator 使用 v1.0.0 版本。
+
+PD、TiDB 和 TiKV 实例数均为 3 个。各组件分别作了如下配置，未配置部分使用默认值。
+
+PD:
+
+```toml
+[log]
+level = "info"
+[replication]
+location-labels = ["region", "zone", "rack", "host"]
+```
+
+TiDB:
+
+```toml
+[log]
+level = "error"
+[prepared-plan-cache]
+enabled = true
+[tikv-client]
+max-batch-wait-time = 2000000
+```
+
+TiKV:
+
+```toml
+log-level = "error"
+[server]
+status-addr = "0.0.0.0:20180"
+grpc-concurrency = 6
+[readpool.storage]
+normal-concurrency = 10
+[rocksdb.defaultcf]
+block-cache-size = "14GB"
+[rocksdb.writecf]
+block-cache-size = "8GB"
+[rocksdb.lockcf]
+block-cache-size = "1GB"
+[raftstore]
+apply-pool-size = 3
+store-pool-size = 3
+```
+
+### TiDB 数据库参数配置
+
+```
+set global tidb_hashagg_final_concurrency=1;
+set global tidb_hashagg_partial_concurrency=1;
+set global tidb_disable_txn_auto_retry=0;
+```
+
+### 硬件配置
+
+#### 机器型号
+
+在单可用区测试中，我们选择了如下型号机器：
+
+| 组件     | 实例类型       | 数量  |
+| :---     | :---------     | :---- |
+| PD       | n1-standard-4  | 3     |
+| TiKV     | c2-standard-16 | 3     |
+| TiDB     | c2-standard-16 | 3     |
+| Sysbench | c2-standard-30 | 1     |
+
+在多可用区相对单可用区性能对比中，因为测试时 (2019.08) GCP 的 Region 均没有 3 个可用区可同时提供 c2 机器，我们选择了如下机器型号进行测试：
+
+| 组件     | 实例类型       | 数量  |
+| :---     | :---------     | :---- |
+| PD       | n1-standard-4  | 3     |
+| TiKV     | n1-standard-16 | 3     |
+| TiDB     | n1-standard-16 | 3     |
+| Sysbench | n1-standard-16 | 3     |
+
+在高并发读测试中，压测端 sysbench 对 CPU 需求较高，需选择多核较高配置型号，避免压测端成为瓶颈。
+
+注：GCP 不同的区域可用机型不同。同时测试中发现磁盘性能表现也存在差异，我们统一在 us-central1 中申请机器测试。
```

Comment:
Use markdown quote?

Reply:
done in https://github.com/pingcap/docs-cn/pull/1803/commits/a6b9cbc56d0f7a04c0b784b75db9fabcd54e38b3, please check